src/ucp/wireup/select.c

```diff
@@ -1489,9 +1555,15 @@ ucp_wireup_add_am_bw_lanes(const ucp_wireup_select_params_t *select_params,
     for (lane_desc_idx = 0; lane_desc_idx < select_ctx->num_lanes; ++lane_desc_idx) {
         if (select_ctx->lane_descs[lane_desc_idx].lane_types &
                 UCS_BIT(UCP_LANE_TYPE_AM)) {
-            /* do not continue searching since we found AM lane (and there is
-             * only one AM lane) */
-            am_lane = lane_desc_idx;
+            /* Do not continue searching since we found AM lane (and there is
+             * only one AM lane). AM lane is also used as a 1st AM_BW lane. */
```

Comment:
i don't think it has to be

Reply:
why? Also it's existing functionality, do we need to change it?

Reply:
@yosefe, do we need to avoid using AM lane as AM_BW lane? If we do, I'd target it for another PR, as it is important change which is unrelated to these bugfixes